package.json

```diff
@@ -64,6 +64,7 @@
     "moment": "2.10.6",
     "moment-timezone": "^0.4.0",
     "morgan": "1.2.0",
+    "ms": "0.7.1",
```

Comment:
> they're both overkill for our needs to set a value in ms

yup, I agree on this.

> Is the idea that we will start using `ms` everywhere that we set values in ms?

It is at least for me.

> But then we need to promote its use across calypso. 

If it's added we should encourage to use it. I think it worth.